src/caja-navigation-window-menus.c

```diff
@@ -819,6 +820,22 @@ action_tab_change_action_activate_callback (GtkAction *action, gpointer user_dat
     }
 }
 
+static gboolean
+on_key_press (GtkWidget *widget, GdkEventKey *event, gpointer user_data)
+{
+    if (event->keyval == GDK_KEY_ISO_Left_Tab && event->state & GDK_CONTROL_MASK)
+    {
+        action_tabs_previous_callback(NULL, user_data);
+        return TRUE;
+    }
+    if (event->keyval == GDK_KEY_Tab && event->state & GDK_CONTROL_MASK)
```

Comment:
Should this be else if statement? One or the other not both of them. I'm new so this project?
Also like this personalty
 if (event->state & GDK_CONTROL_MASK)
{
     switch (event->keyal)
.....
}